code/modules/reagents/newchem/newchem_drugs.dm

```diff
@@ -0,0 +1,502 @@
+#define SOLID 1
+#define LIQUID 2
+#define GAS 3
+
+#define REM REAGENTS_EFFECT_MULTIPLIER
+
```

Comment:
Should probably separate some of the more controversial drugs and their intermediates into separate files so they can be unticked as needed.